src/ucp/core/ucp_thread.h

```diff
@@ -44,21 +44,26 @@ typedef struct ucp_mt_lock {
 
 #define UCP_THREAD_IS_REQUIRED(_lock_ptr) \
     ((_lock_ptr)->mt_type)
-#define UCP_THREAD_LOCK_INIT(_lock_ptr)                                 \
-    {                                                                   \
-        if ((_lock_ptr)->mt_type == UCP_MT_TYPE_MUTEX) {                \
-            pthread_mutex_init(&((_lock_ptr)->lock.mt_mutex), NULL);    \
-        } else {                                                        \
-            ucs_spinlock_init(&((_lock_ptr)->lock.mt_spinlock));        \
-        }                                                               \
-    }
-#define UCP_THREAD_LOCK_FINALIZE(_lock_ptr)                             \
-    {                                                                   \
-        if ((_lock_ptr)->mt_type == UCP_MT_TYPE_MUTEX) {                \
-            pthread_mutex_destroy(&((_lock_ptr)->lock.mt_mutex));       \
-        } else {                                                        \
-            ucs_spinlock_destroy(&((_lock_ptr)->lock.mt_spinlock));     \
-        }                                                               \
+#define UCP_THREAD_LOCK_INIT(_lock_ptr)                              \
```

Comment:
could you pls fix alignment of `\`, i.e. it should be:
```
define UCP_THREAD_LOCK_INIT(_lock_ptr) \
    do { \
...
```
instead of:
```
define UCP_THREAD_LOCK_INIT(_lock_ptr) \
    do {                               \
...
```